metriq_gym/benchmarks/benchmark.py

```diff
@@ -1,17 +1,31 @@
 import argparse
-from typing import Any
 
-from metriq_gym.job_manager import JobManager
+from pydantic import BaseModel
+from dataclasses import dataclass
+
+from qbraid import QuantumDevice, ResultData
+
+
+@dataclass
+class BenchmarkData:
+    provider_job_id: list[str]
 
 
 class Benchmark:
```

Comment:
We don't have to do this in this diff, but the linter (I think rightfully) complains that the args in the dispatch_* and poll_* functions are not used. If the class was abstract, I think it would make that linter happy, make the relationship of the class clear, and enforce that dispatch and poll are implemented for subsequent benchmarks.